app/views/groups/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class='text-center p-3'>
+<h1>All Transactions for <b><%= @owned_trans.name %></b></h1>
+<span class="badge badge-lg bg-primary ">total: <%= @owned_trans.trans.sum(:amount) %>$</span>
+</div>
+<% if @owned_trans.trans.length < 1 %>
+<div class='text-center p-3'>
+<p class='text-center text-md'>You have no transactions please click the add button</p>
+</div>
+<% end  %>
+  
+  
+  <ol class="list-group list-group-numbered p-3">
+  <% @owned_trans.trans.order(created_at: :desc).each do |card| %>
+
+  <li class="list-group-item d-flex justify-content-between align-items-start">
+    <div class="ms-2 me-auto">
+      <div class="fw-bold">    <%= card.name %></div>
+      <%= distance_of_time_in_words(card.created_at, DateTime.now) %> ago
+    </div>
+    <span class="badge bg-primary rounded-pill"><%= card.amount %> $</span>
+  </li>
+
+
+<% end %>
+</ol>
+
+<div class="d-flex align-content-center justify-content-center">
+<%= link_to  "Add", new_tran_path , class:'btn btn-success'%>
+</div>
```

Comment:
- [ ] Kindly design your transaction page to match the template. We should maintain the page layout and the cards of the transaction should also be styled accordingly.

  Your transaction page | Design
  :---:|:---:
  ![diel](https://user-images.githubusercontent.com/54848363/227490676-65beb67f-9a64-41ad-a47e-7e4f8066869b.jpg) | ![hsyw](https://user-images.githubusercontent.com/54848363/227490661-4c405db7-f95d-49e7-bbca-57687a55f03f.jpg)